instrumentation/opentelemetry-instrumentation-system-metrics/src/opentelemetry/instrumentation/system_metrics/__init__.py

```diff
@@ -225,27 +240,51 @@ def _instrument(self, **kwargs):
         #     value_type=int,
         # )
 
-        if "system.disk.io" in self._config:
+        if "system.disk.io.read" in self._config:
             self._meter.create_observable_counter(
-                name="system.disk.io",
-                callbacks=[self._get_system_disk_io],
-                description="System disk IO",
+                name="system.disk.io.read",
+                callbacks=[self._get_system_disk_io_read],
+                description="System disk IO read",
```

Comment:
Please use the precise descriptions for each of these instruments as defined by the specification.

Reply:
@lzchen Appreciate the guidance. This is the spec I'm working off. Should I assume that metrics where the description is empty, I should leave it empty as well?

https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/semantic_conventions/system-metrics.md#systemdisk---disk-controller-metrics

Reply:
I believe I have addressed this with the most recent commits